main.go

```diff
@@ -13,9 +13,12 @@ import (
 	"github.com/NBISweden/sda-cli/helpers"
```

Comment:
In many `import` cases, packages such as `helpers` that are included in the repository are imported as remote repos. I don't quite understand the benefit of it. May someone explain it @pahatz @norling @dbampalikis ?

Reply:
Sorry, I didn't realized that the project name is defined as `github.com/NBISweden/sda-cli`  in https://github.com/NBISweden/sda-cli/blob/main/go.mod#L1. So this is a local importing.